shell/apps/common/firebase-config.js

```diff
@@ -8,7 +8,8 @@ Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 */
 
-import firebase from '../components/firebase.4.2.0.js';
+// firebase runtime (customized by sjmiles@ for import-ability)
```

Comment:
My feeling is that this will never matter, but I know Walt had some concerns too. I added `firebase.4.2.0.readme` in this folder to document the changes. I'm happy to do whatever else you think would de-risk this situation.